tools/reduce/Cargo.toml

```diff
@@ -23,7 +23,7 @@ edition = "2021"
 [dependencies]
 autocxx-engine = { version = "=0.16.0", path="../../engine" }
 autocxx-parser = { version = "=0.16.0", path="../../parser", features = [ "reproduction_case" ] }
-clap = "~2.33.0"
+clap = { version = "3.1.2", features = ["cargo", "derive"] }
```

Comment:
Same question about the additional features - probably not needed?

Reply:
yup, probably not needed!